### DISCUSSION TITLE
jssip + vite + pinia，unable set jssip UA to state.

### DISCUSSION THREAD

**Discussion by Abbyzhoum** [vuejs/pinia#1161] [Help and Questions] [answered]
### Reproduction

none

### Steps to reproduce the bug

Other states can be set to the state normally, but the UA state in jssip cannot be set to the state, and the setting reports an error.
Below is the code block.

![image](https://user-images.githubusercontent.com/38661489/159233745-d1facfba-e87b-420d-879d-21ff82a8f7f9.png)

![image](https://user-images.githubusercontent.com/38661489/159233823-1db78bd0-ecc5-46ab-803f-df48f5a40ad0.png)

### Expected behavior

The UA can be set to the state in the store, and the UA value can be retrieved from the store.

### Actual behavior

The UA state value cannot be set to the state, an error is reported。

Troubleshooting：
`
VM13502:1 [Hook] Error in async event handler for custom-inspector:send-state with args: (2) ['pinia', {…}]
(anonymous) @ VM13502:1
Promise.catch (async)
emit @ VM13502:1
sendInspectorState @ backend.js:488
store.$subscribe.detached @ pinia.esm-browser.js:779
(anonymous) @ pinia.esm-browser.js:1335
callWithErrorHandling @ runtime-core.esm-bundler.js:155
callWithAsyncErrorHandling @ runtime-core.esm-bundler.js:164
job @ runtime-core.esm-bundler.js:1779
triggerEffects @ reactivity.esm-bundler.js:359
trigger @ reactivity.esm-bundler.js:329
set2 @ reactivity.esm-bundler.js:484
set value @ reactivity.esm-bundler.js:1069
set2 @ reactivity.esm-bundler.js:470
set @ pinia.esm-browser.js:884
setId @ index.ts:79
(anonymous) @ pinia.esm-browser.js:1295
store.<computed> @ pinia.esm-browser.js:887
register @ m.register.ts:47
UARegister @ root.vue:73
(anonymous) @ root.vue:64
validateField @ form.vue:126
await in validateField (async)
validate @ form.vue:91
checkRegisterParams @ root.vue:61
_createVNode.onClick._cache.<computed>._cache.<computed> @ root.vue:21
callWithErrorHandling @ runtime-core.esm-bundler.js:155
callWithAsyncErrorHandling @ runtime-core.esm-bundler.js:164
emit$1 @ runtime-core.esm-bundler.js:718
(anonymous) @ runtime-core.esm-bundler.js:7232
handleClick @ button.vue:141
callWithErrorHandling @ runtime-core.esm-bundler.js:155
callWithAsyncErrorHandling @ runtime-core.esm-bundler.js:164
invoker @ runtime-dom.esm-bundler.js:366
17:19:57.055 VM13502:1 TypeError: 'get' on proxy: property 'uri' is a read-only and non-configurable data property on the proxy target but the proxy did not return its actual value (expected '[object Object]' but got '[object Object]')
`

![image](https://user-images.githubusercontent.com/38661489/159233461-1b015d87-fccf-464b-91bd-d91bb9cf425e.png)

**Comment by posva** [comment, marked as answer]
You probably need to call markRaw(storeInterface)

**Reply by Abbyzhoum** [reply]
Thanks ！It worked！👍🏻
